Lib/test/crashers/null_code_obj.py

```diff
@@ -0,0 +1,36 @@
+import platform
+
+"""
+This execute a payload code object which crashes the interpeter for different
+python versions. Currently there is no bytecode verifier to prevent those
+crashes related to malformed bytecode
+"""
+
+
+def crash_27():
+    exec((lambda f:f).func_code.__class__(0,0,0,0,"\x01",(),(),(),"","",0,""))
+
+
+def crash_36():
+    exec((lambda f:f).__code__.__class__(0,0,0,0,0,b"\x01",(),(),(),"","",0,b""))
+
+
+def crash_38():
+    exec((lambda f: f).__code__.__class__(0,0,0,0,0,0,b"\x01",(),(),(),"","",0,b""))
+
+
+def crash():
+    locals()['crash_27'] = crash_27
+    locals()['crash_36'] = crash_36
+    locals()['crash_37'] = crash_36
+    locals()['crash_38'] = crash_38
+    locals()['crash_39'] = crash_38
+    locals()['crash_310'] = crash_38
```

Comment:
The test suite only ever runs on the current version of python. I think you can restrict this to just crash_38 code, since at best this would be backported to 3.8 and 3.9.

2.7 is no longer maintained, so that can definitely be removed.

Reply:
Thanks @ericvsmith for your suggestions. I have removed the 2.7 part and keep only the crash_38 for master 🤝